test/core/miner/sealing_test.cpp

```diff
@@ -426,6 +426,7 @@ namespace fc::mining {
   TEST_F(SealingTest, MarkForUpgradeSeveralPieces) {
     UnpaddedPieceSize piece_size(127);
     PieceData piece("/dev/random");
+    PieceData piece2("/dev/random");
```

Comment:
```suggestion
    const UnpaddedPieceSize piece_size(127);
    const PieceData piece("/dev/random");
    const PieceData piece2("/dev/random");
```

Reply:
Fixed

Reply:
```
    const PieceData piece("/dev/random");
    const PieceData piece2("/dev/random");
```

This is wrong because on line 472-475: ``